source/back.tex

```diff
@@ -22,8 +22,8 @@ \chapter{Bibliography}
 
 \clearpage
 \renewcommand{\glossaryname}{Cross references}
-\renewcommand{\preglossaryhook}{This annex lists each section label and the
-corresponding section number and page number, in alphabetical order by label.\\}
+\renewcommand{\preglossaryhook}{This annex lists each subclause label and the
+corresponding subclause number and page number, in alphabetical order by label.\\}
```

Comment:
These ones mean "clause or subclause". We could use "[...] lists each clause or subclause label and [...]" but perhaps there's a better way to say that?